avail/src/test/kotlin/avail/test/SerializerTest.kt

```diff
@@ -124,7 +124,7 @@ class SerializerTest
 	{
 		val semaphore = Semaphore(0)
 		val projectDirectory = File(".").absolutePath
-			.replace("/avail-core", "")
+			.replace("/avail", "")
```

Comment:
See previous comment (i.e., `.replace("/avail/", "/")`)

Reply:
Turns out this broke everything. The directory never had a full `/avail/` only `/avail` so the proper replace was not happening. So I reverted all these changes.